src/extensions/vectorstores/OpenSearchVectorUtils.py

```diff
@@ -74,12 +80,16 @@
 def create_authentication_params(context):
     username = context.getProperty(USERNAME).getValue()
     password = context.getProperty(PASSWORD).getValue()
+    certificate_path = context.getProperty(CERTIFICATE_PATH).getValue()
 
-    params = {"verify_certs": "true"}
```

Comment:
@mark-bathori is there a particular reason for removing this `verify_certs` property? Does it somehow conflict with specifying a custom certificate? It seems like it should remain in place and only be changed when a custom certificate is configured.

Reply:
Thanks @exceptionfactory for the review. I removed the flag because it is automatically set based on the provided url. If the url is using **https** scheme then the **use_ssl** property will be set to true in the [utils.py](https://github.com/opensearch-project/opensearch-py/blob/main/opensearchpy/client/utils.py#L70) and because of that value the **verify_certs** will be set to true in the [http_urllib3.py](https://github.com/opensearch-project/opensearch-py/blob/main/opensearchpy/connection/http_urllib3.py#L189).

Reply:
Thanks for clarification and background @mark-bathori, sounds good.